test/api_test.rb

```diff
@@ -51,6 +51,7 @@ class JobWithTags
       assert_equal 0, s.failed
       assert_equal 0, s.enqueued
       assert_equal 0, s.default_queue_latency
+      assert_kind_of Float, s.default_queue_latency
```

Comment:
`assert_equal` doesn't distinguish float 0.0 and integer 0, so I have added class assertion